turbinia/workers/partitions.py

```diff
@@ -130,7 +141,27 @@ def _ProcessPartition(self, path_spec):
 
       child_path_spec = child_path_spec.parent
 
+    # Is partition important based on filesystem?
+    if path_spec.type_indicator not in ('APFS', 'EXT', 'HFS', 'NTFS', 'TSK',
+                                        'XFS'):
+      important = False
+      log.info(
+          'Marking partition {0:s} unimportant (filesystem {1:s})'.format(
+              location, path_spec.type_indicator))
+
+    # Is partition important based on size? (100M or larger)
+    if partition_size and partition_size < 104857600:
+      important = False
+      log.info(
+          'Marking partition {0:s} unimportant (size {1!s} bytes)'.format(
```

Comment:
Could also be a little more explicit here and say "size foo < size bar".

Reply:
Done